@xen-orchestra/web/src/components/host/network/HostPifSidePanel.vue

```diff
@@ -41,211 +41,110 @@
               </UiComplexIcon>
               <a href="">{{ networkNameLabel }}</a>
-              -->
               <span v-tooltip class="value text-ellipsis">{{ network?.name_label }}</span>
             </template>
-            <template v-if="network?.name_label" #addons>
-              <VtsCopyButton :value="network.name_label" />
-            </template>
-          </VtsCardRowKeyValue>
+-->
+          </UiLabelValue>
           <!-- DEVICE -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('device') }}
-            </template>
-            <template #value>
-              {{ pif.device }}
-            </template>
-            <template #addons>
-              <VtsCopyButton :value="pif.device" />
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue :label="t('device')" :value="pif.device" :copy-value="pif.device" ellipsis />
           <!-- PIF STATUS -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ pif.isBondMaster ? t('bond-status') : t('pif-status') }}
-            </template>
+          <UiLabelValue :label="pif.isBondMaster ? t('bond-status') : t('pif-status')" ellipsis>
             <template #value>
               <VtsStatus :status />
             </template>
-          </VtsCardRowKeyValue>
+          </UiLabelValue>
           <!-- PHYSICAL INTERFACE STATUS -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('physical-interface-status') }}
-            </template>
+          <UiLabelValue :label="t('physical-interface-status')" ellipsis>
             <template #value>
               <VtsStatus :status="physicalInterfaceStatus" />
             </template>
-          </VtsCardRowKeyValue>
+          </UiLabelValue>
           <!-- VLAN -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('vlan') }}
-            </template>
-            <template #value>
-              {{ pif.vlan === -1 ? t('none') : pif.vlan }}
-            </template>
-            <template v-if="pif.vlan !== -1" #addons>
-              <VtsCopyButton :value="String(pif.vlan)" />
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue
+            :label="t('vlan')"
+            :value="pif.vlan === -1 ? t('none') : String(pif.vlan)"
+            :copy-value="pif.vlan === -1 ? undefined : String(pif.vlan)"
+            ellipsis
+          />
           <!-- TAGS -->
-          <VtsCardRowKeyValue>
-            <template #key>
-              {{ t('tags') }}
-            </template>
-            <template #value>
-              <UiTagsList class="value">
-                <UiTag v-for="tag in network?.tags" :key="tag" accent="info" variant="secondary">
-                  {{ tag }}
-                </UiTag>
-              </UiTagsList>
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue :label="t('tags')" :value="network?.tags" ellipsis />
```

Comment:
```suggestion
          <UiLabelValue :label="t('tags')" :value="network?.tags" />
```
I don't think there are any ellipsis for tags in sidepanel